tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2840,73 +2852,141 @@ NodeDef GetAddNNodeDef(const std::vector<string>& input_names, DataType dtype) {
   return op.operation.node()->def();
 }
 
-template <DataType dtype>
-void TestAddN(OpConverterTest* test) {
-  typedef typename EnumToDataType<dtype>::Type CType;
+struct AddNTestParams {
+  std::vector<float> input_value;
+  std::vector<string> input_names;
+  std::vector<int> expected_dim;
```

Comment:
Can we call this "dimensions", since it is the dimensions for the inputs and the output?